presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+import static com.facebook.presto.spi.statistics.StatisticsValue.unknownStatistics;
+import static java.util.Objects.requireNonNull;
+
+public final class ColumnStatistics
+{
+    public static final ColumnStatistics EMPTY_STATISTICS = ColumnStatistics.builder().build();
+
+    private final StatisticsValue dataSize;
+
+    private ColumnStatistics(StatisticsValue dataSize)
+    {
+        this.dataSize = requireNonNull(dataSize, "dataSize can not be null");
+    }
+
+    public StatisticsValue getDataSize()
+    {
+        return dataSize;
+    }
+
+    public static Builder builder()
+    {
+        return new Builder();
+    }
+
+    public static final class Builder
```

Comment:
On the other hand we will add new stuff soon here. So I am opting for leaving builder here. 
I will add the nullcheck in setter.